CMakeLists.txt

```diff
@@ -93,10 +97,17 @@ endif (${CMAKE_SYSTEM_PROCESSOR} MATCHES "^arm.*$")
 #
 if (MACOSX)
   set (CMAKE_INSTALL_LIBDIR "lib")
+  set (CMAKE_INSTALL_BINDIR "/sbin")
```

Comment:
Can you rename that from `CMAKE_INSTALL_BINDIR` to `CMAKE_MOUNT_INSTALL_BINDIR`, since it only changes the install location of the mount helper, right?